src/utils/helpers/style-helper/style-helper.js

```diff
@@ -0,0 +1,6 @@
+import { THEMES } from '../../../style/themes';
+
+// eslint-disable-next-line import/prefer-default-export
+export function isClassic({ name }) {
```

Comment:
Why don't we `export default` if it is only one export here?

Reply:
It also remove eslint warning

Reply:
I have used a named export to to allow other helper functions to be added as there are a lot of examples where the people have used functions that can be moved into a more global scope